### DISCUSSION TITLE
What’s Next For ToonSquid?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#543] [General]
It's been 5 months since the last big update, is 1.11 next, or is 2.0 on the horizon? I liked as many suggestions as I could in hopes that they'd be added to ToonSquid in the future. 

I hope we get a fix to the out-of-pegs feature, so we can edit it as we go without resetting it. Some animated effects, such as Gaussian blur and motion blur, would be appreciated also.

**Comment by Lioncub901** [comment]
I feel like if he is making 2.0. Keiwan he should maybe take a year to develop it, so that he can make it into a massive update with a lot of the features we want since it is a major update. About the blurs. I would prefer it to be a node vfx system like in OpenToonz to make it interesting.

**Comment by keiwando** [comment]
I'm working on the 2.0 update at the moment and it's a rather big one, so it's going to take significantly more time than the previous updates.

**Reply by DWrightAnimation** [reply]
Thank you for clarifying that for us. Could you give us a hint of what it will have in it?

**Reply by Lioncub901** [reply]
No spoilers we need to be surprised, if you know what I mean.

**Reply by DWrightAnimation** [reply]
I hope we get some kind of inverse kinematics and bone features in 2.0. If not, maybe some update afterwards might have it. I'd also hope some improvements to the out-of-pegs feature and a fixed onion skin toggle, so we can get an onion skin of any frame, regardless of how far we are from it. Though, I would like to know when we should expect the update to launch.

**Reply by Dwork94** [reply]
Is there a kickstarter for this, would like too push big things?

**Reply by DWrightAnimation** [reply]
> Is there a kickstarter for this, would like too push big things?

Honestly, I think this is a good idea, that way ToonSquid can get more and more powerful, and probably beat out the competition.

**Reply by trinhpxl** [reply]
yo omggg good luck keiwando on the update, thank u for ur hard work sir 🫡🫡🫡 we all cant wait to see it :D !!

**Reply by Lioncub901** [reply]
I kind of want there to be vfx(node system) and adjustments.

**Reply by DinoAwakening** [reply]
Keiwando is just amazing. ToonSquid already is on top of most animation apps out there. Way more features, and surprisingly affordable. This man singlehandely drew a new standard when it comes to artist softwares. Impressive.

Can’t wait to see what’s coming up next! :D

**Reply by Illustr8rGIT** [reply]
Yeah, I vastly prefer it over Procreate Dreams.

**Reply by Cjjuntti** [reply]
One thing I’d love to see is more options for the camera layer, like being able to offset layers slightly from each other to create an illusion of depth. I think you can technically do that by keyframing each individual layer, but I’d rather be able to do it with the camera layer, and also the ability to take some layers off the camera, so they don’t move when the camera moves.

**Reply by steryo** [reply]
That's amazing news because it sure is great at this point. As a professional animator, I established a new workflow over ToonSquid and can't wait for new features to expand it more.

**Reply by TempleLionStudios** [reply]
Keiwan my man! So glad to hear this! You never disappoint! Godspeed with the update dev. 🥳2.0!

**Comment by Illustr8rGIT** [comment]
Do you envision a Mac desktop version somewhere down the road? I love the IOS version but I’d like to use it on a larger screen, and Apple has yet to make an iPad the size of a restaurant tray…

**Reply by Lioncub901** [reply]
I think he did say that he may make a Mac Version, but later if a lot demand.
![IMG_2695](https://github.com/keiwando/toonsquid/assets/88890833/35a03543-502e-494b-b00d-fbdf105e6cff)

**Reply by Illustr8rGIT** [reply]
As long as he hasn’t completely written off the idea. I’d happily pay for it if it’s not subscription based.

**Reply by Cat-Butler** [reply]
I'd be a +1 for the Mn (M1,M2,M3) version and, like @Illustr8rGIT, happy to pay for it separately.

**Reply by Dwork94** [reply]
I'll honestly pay anything but a subscription, especially if it includes a quick library like Adobe and possible rigging. Procreate dreams is worse than just animating in regular procreate honestly haha., animation pro is ok but there in app drawing software is trash, and the creator doesn't see it and made a tutorial for everything besides how to draw a character

**Reply by Illustr8rGIT** [reply]
Well, that and the fact that it's crashy. And the UI is meh. And the default figures all look crappy and amateurish; the main mascot looks like Barney Rubble's head on Peter Griffin's body. (Yes, I bought it, too.) A positive thing about Animation Pro: it's about 90% there in terms of the inverse kinematics feature, but it needs the equivalent of Moho's anchor bones to, well, ANCHOR the character somewhere other than its center. If there's any other inexpensive iPad-based 2D animation apps that have an IK feature I'm unaware of them.

Anyway, I had emailed the creator a while back and apparently he'd had a cardiac arrest and a subsequent brain condition, so in addition to whatever other reasons he might have, he's only working on Animation Pro as a hobby and not as a main project.

**Reply by Dwork94** [reply]
That's terrible but yea the UI, and in app drawing is the deal breaker for me. I just want to draw like procreate save as a symbol and pull all my symbols up on the right side and either rig or drag and drop a symbol every few frames like adobe or maya but without the over done frustration. I want to make cartoons, not have to really think about the software or have to open the symbol library or set quick symbols every scenes like the TS mouth tutorial, when I could open the mouths for my character and select the one I need nice and organized or maybe select from my 20 different character poses I already have drawn.

**Reply by Mirakelpung** [reply]
> Procreate dreams is worse than just animating in regular procreate honestly haha

Tell me about it. I was to excited about that program because I've wanted Procreate with a timeline and Audio imports for ages and when we finally got it the UI is an absolute mess and doesn't feel like standard Procreate at all, and it's missing basic key features like a lasso tool. I also don't think it's able to create scenes like Toonsquid
